modules/library/src/main/scala/zio/elasticsearch/Routing.scala

```diff
@@ -63,6 +72,17 @@ object Routing extends Newtype[String] {
         }
     }
 
+    implicit val deleteByQueryWithRouting: WithRouting[DeleteByQuery] = new WithRouting[DeleteByQuery] {
+      def withRouting[A](
+        request: ElasticRequest[A, DeleteByQuery],
+        routing: Routing
+      ): ElasticRequest[A, DeleteByQuery] =
+        request match {
+          case Map(r, mapper)          => Map(withRouting(r, routing), mapper)
+          case r: DeleteByQueryRequest => r.copy(routing = Some(routing))
```

Comment:
Should we put `Map` at the bottom? Not sure about consistency with existing code?

Reply:
That is the order for all routing and refresh instances, so we would have to change it there too

Reply:
Mea culpa 🙋‍♂️